### PR TITLE
add fuzzy search app

### PR DESCRIPTION
**Pull request by HarshCasper**

### CODE REVIEW

File: data/developerhub/applications.json [hunk 1]
```diff
@@ -181,6 +181,19 @@
...
+      "title": "Search application with Lambda, Kinesis, Firehose, ElasticSearch, S3",
+      "description": "Fuzzy Search application demonstrates how to fuzzy-search a movie database using various AWS services, deployed using Terraform and AWS CLI on LocalStack",
+      "url": "https://github.com/localstack/sample-fuzzy-movie-search-lambda-kinesis-elasticsearch",
+      "teaser": "https://raw.githubusercontent.com/localstack/sample-fuzzy-movie-search-lambda-kinesis-elasticsearch/main/images/system_overview.png",
+      "services": ["s3", "lmb", "kns", "es"],
+      "deployment": ["terraform"],
+      "platform": ["python", "javascript"],
+      "tags": ["serverless", "event-driven-architecture"],
+      "complexity": ["intermediate"],
+      "pro": false,
```
Comment (steffyP) [inline data/developerhub/applications.json:195]:
```suggestion
      "pro": true,
```

File: data/developerhub/applications.json [hunk 2]
```diff
@@ -181,6 +181,19 @@
       "complexity": ["intermediate"],
       "pro": true,
       "cloudPods": false
+    },
+    {
+      "title": "Search application with Lambda, Kinesis, Firehose, ElasticSearch, S3",
+      "description": "Fuzzy Search application demonstrates how to fuzzy-search a movie database using various AWS services, deployed using Terraform and AWS CLI on LocalStack",
+      "url": "https://github.com/localstack/sample-fuzzy-movie-search-lambda-kinesis-elasticsearch",
+      "teaser": "https://raw.githubusercontent.com/localstack/sample-fuzzy-movie-search-lambda-kinesis-elasticsearch/main/images/system_overview.png",
+      "services": ["s3", "lmb", "kns", "es"],
```
Comment (steffyP) [inline data/developerhub/applications.json:190]: firehose is missing from the services list, so we won't be able to filter it. 
or do we use the term "kinesis" for both  kinesis firehose + kinesis data stream?

**Review by steffyP — COMMENTED** [review]
Thanks for adding the sample 🙏 🥳 
Only two remarks (see comments)

**Review by steffyP — APPROVED** [review]

File: data/developerhub/applications.json (hunk 2 above)
Comment (HarshCasper) [reply data/developerhub/applications.json:190]: Let's use the term "kinesis" for both kinesis firehose + kinesis data stream (for now!) 👍

**Pull request merged**